### ISSUE TITLE
CPU not known on 0.7.4

### ISSUE THREAD

**Issue by vks** [VcDevel/Vc#6]
When using Vc 0.7.4, running cmake yields:

```
[...]
CMake Warning at cmake/OptimizeForArchitecture.cmake:110 (message):
  Your CPU (family 6, model 60) is not known.  Auto-detection of optimization
  flags failed and will use the 65nm Core 2 CPU settings.
Call Stack (most recent call first):
  cmake/OptimizeForArchitecture.cmake:159 (AutodetectHostArchitecture)
  cmake/VcMacros.cmake:382 (OptimizeForArchitecture)
  CMakeLists.txt:105 (vc_set_preferred_compiler_flags)


-- Detected CPU: merom
[...]
```

This has been fixed on master. It should probably be backported.

```
$ cat /proc/cpuinfo
processor	: 1
vendor_id	: GenuineIntel
cpu family	: 6
model		: 60
model name	: Intel(R) Core(TM) i7-4710HQ CPU @ 2.50GHz
stepping	: 3
microcode	: 0x1a
cpu MHz		: 2838.671
cache size	: 6144 KB
physical id	: 0
siblings	: 8
core id		: 0
cpu cores	: 4
apicid		: 1
initial apicid	: 1
fpu		: yes
fpu_exception	: yes
cpuid level	: 13
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf eagerfpu pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm ida arat epb pln pts dtherm tpr_shadow vnmi flexpriority ept vpid fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt
bugs		:
bogomips	: 4990.20
clflush size	: 64
cache_alignment	: 64
address sizes	: 39 bits physical, 48 bits virtual
power management:
[...]
```

**Comment by mkret2** [comment]
Actually, model 60 is already in the 0.7 branch in git. So this is fixed.

**Comment by vks** [comment]
BTW, you can close an issue from a commit by having `Fixes #6` in the commit message.
